pyscal/factory.py

```diff
@@ -271,6 +312,29 @@ def create_water_oil(
             wateroil.add_simple_J_petro(**params_simple_j_petro)
         elif set(WO_NORM_J).issubset(set(params_norm_j)):
             wateroil.add_normalized_J(**params_norm_j)
+        elif set(WO_LET_PC_PD).issubset(set(params_let_pc_pd)):
+            params_let_pc_pd["Lp"] = params_let_pc_pd.pop("lpow")
```

Comment:
Do you need to rename the dictionary entries? If possible, it could be beneficial to keep `"lpow"`

Reply:
It was similar set-up for the LET relperm function, so that why I renamed them - to make it consistent.

Reply:
good, then it is prorably fine

Reply:
I think this is mainly because Subhi want to support all fluid system (oil-water, oil-gas, and gas-water) and at the same time fit with existing function in pyscal